2022-01-09 18:17:12,313 INFO    MainThread:17630 [wandb_setup.py:_flush():71] setting env: {}
2022-01-09 18:17:12,314 INFO    MainThread:17630 [wandb_init.py:_log_setup():371] Logging user logs to /content/tbcnn/wandb/offline-run-20220109_181712-lptfuf9v/logs/debug.log
2022-01-09 18:17:12,314 INFO    MainThread:17630 [wandb_init.py:_log_setup():372] Logging internal logs to /content/tbcnn/wandb/offline-run-20220109_181712-lptfuf9v/logs/debug-internal.log
2022-01-09 18:17:12,314 INFO    MainThread:17630 [wandb_init.py:init():404] calling init triggers
2022-01-09 18:17:12,314 INFO    MainThread:17630 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-01-09 18:17:12,314 INFO    MainThread:17630 [wandb_init.py:init():460] starting backend
2022-01-09 18:17:12,314 INFO    MainThread:17630 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-09 18:17:12,328 INFO    MainThread:17630 [backend.py:ensure_launched():216] starting backend process...
2022-01-09 18:17:12,336 INFO    MainThread:17630 [backend.py:ensure_launched():222] started backend process with pid: 17640
2022-01-09 18:17:12,339 INFO    MainThread:17630 [wandb_init.py:init():469] backend started and connected
2022-01-09 18:17:12,350 INFO    MainThread:17630 [wandb_init.py:init():533] updated telemetry
2022-01-09 18:17:12,391 INFO    MainThread:17630 [wandb_init.py:init():606] starting run threads in backend
2022-01-09 18:17:16,204 INFO    MainThread:17630 [wandb_run.py:_console_start():1810] atexit reg
2022-01-09 18:17:16,204 INFO    MainThread:17630 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-09 18:17:16,205 INFO    MainThread:17630 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-09 18:17:16,206 INFO    MainThread:17630 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-09 18:17:16,207 INFO    MainThread:17630 [wandb_init.py:init():633] run started, returning control to user process
2022-01-09 18:18:13,534 INFO    MainThread:17630 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2022-01-09 18:18:13,536 INFO    MainThread:17630 [wandb_run.py:_restore():1752] restore
2022-01-09 18:18:15,941 INFO    MainThread:17630 [wandb_run.py:_wait_for_finish():1912] got exit ret: 
2022-01-09 18:18:16,044 INFO    MainThread:17630 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
local_info {
}

